.github/workflows/main.yml

```diff
@@ -9,11 +9,8 @@ jobs:
       run: | 
         cd frontend
         yarn add eslint -g 
-        yarn add eslint-plugin-react eslint-config-airbnb-typescript  --save-dev
         npx eslint --ext .ts,.tsx src/
         cd ../backend
-        yarn
```

Comment:
I think we have to keep this?

Reply:
![image](https://user-images.githubusercontent.com/43439578/107628029-5abe1900-6c60-11eb-9a7f-7d9bae98112a.png)

What do you mean? It says all checks have passed 🤔 tested with and without yarn and it didn't make a difference (as can be seen in the commits)

Reply:
Oh, looked at the wrong commit I guess. Alright